src/main/java/duke/Event.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * An event task with a description and an occurrence date.
+ */
+public class Event extends Task {
+
+    private LocalDate at;
```

Comment:
Hi Marcus, it was not immediately obvious what at meant. I would like to suggest using a more descriptive name, such as startTime.